docs/guides/aens.md

```diff
@@ -363,20 +363,13 @@ const client = await Universal({ ... }) // init instance of Universal Stamp
 const contractId = 'ct_asd2ks...'
 // AENS name
 const name = 'example.chain'
+// Sign with a specific account
+const onAccount = await client.address()
 
 // this signature will allow the contract to perform a pre-claim on your behalf
-const sig = await client.delegateNamePreclaimSignature(contractId)
+const preClaimSig = await client.createAensDelegationSignature({ contractId }, { onAccount })
 
 // this signature will allow the contract to perform
 // any name related transaction for a specific name that you own
-const sig = await client.delegateNameClaimSignature(contractId, name)
+const aensDelegationSig = await client.createAensDelegationSignature({ contractId, name }, { onAccount })
 ```
-
-Note:
-
-- If required you can pass an additional options object like `{ onAccount: <account> }` to these functions in order to sign with a specific account.
```

Comment:
as you removed this note => is not possible to specify another account for signature creation anymore? otherwise we should keep this specific note :)

and I think it would be good to have it working this way, as almost every function allows that

Reply:
Yes, Absolutely possible to specify another account.

former `delegateSignatureCommon`:  Select one existing account from the stamp to be included in the signature.
new  `delegateSignatureCommon`:  Don't choose any accounts by default unless  `{ onAccount }` in the options.

If you observe the new guide https://github.com/aeternity/aepp-sdk-js/blob/252f5d32d913176a6d0d6c929b3665e4dfcd612f/docs/guides/aens.md#delegate-signature-to-contract-aens-interface It's a must to have `onAccount` option in the methods. It will eliminate accidentally signing the signature from unintended accounts because you need to specifically pass an account field in the options. This may be a limitation given almost every function allows without selecting a specific account. Well, In this case, I have to modify the `delegateSignatureCommon` to choose the default account.

Reply:
ok, got it